tests/phpunit/integration/tests/Admin/Admin.php

```diff
@@ -222,4 +230,23 @@ public function test_prefill_post_title_no_texturize(): void {
 
 		$this->assertSame( 'Story - Test', $result );
 	}
+
+	/**
+	 * @covers ::media_states
+	 */
+	public function test_media_states_no_active_logo(): void {
+		$post   = self::factory()->post->create_and_get( [] );
+		$result = $this->instance->media_states( [], $post );
+		$this->assertSame( [], $result );
+	}
+
+	/**
+	 * @covers ::media_states
+	 */
+	public function test_media_states_with_active_logo(): void {
+		$post = self::factory()->post->create_and_get( [] );
+		update_option( $this->settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO, $post->ID );
```

Comment:
```suggestion
		$this->settings->update_setting( $this->settings::SETTING_NAME_ACTIVE_PUBLISHER_LOGO, $post->ID );
```

Reply:
@spacedmonkey Can you elaborate on this suggestion? We do not do this for any other integration test. All of them directly use `update_option`. Why change now?

Reply:
Code consistency is also important